File: common/src/main/java/juuxel/adorn/client/gui/screen/AdornMenuScreens.java
package juuxel.adorn.client.gui.screen;

import juuxel.adorn.menu.AdornMenus;
import net.minecraft.client.gui.screens.MenuScreens;

public final class AdornMenuScreens {
    public static void register() {
        MenuScreens.register(AdornMenus.DRAWER.get(), DrawerScreen::new);
        MenuScreens.register(AdornMenus.KITCHEN_CUPBOARD.get(), KitchenCupboardScreen::new);
        MenuScreens.register(AdornMenus.TRADING_STATION.get(), TradingStationScreen::new);
        MenuScreens.register(AdornMenus.BREWER.get(), BrewerScreen::new);
    }
}

File: common/src/client/kotlin/juuxel/adorn/client/gui/screen/AdornMenuScreens.kt
package juuxel.adorn.client.gui.screen

import juuxel.adorn.menu.AdornMenus
import net.minecraft.client.gui.screen.ingame.MenuScreens

object AdornMenuScreens {
    fun register() {
        MenuScreens.register(AdornMenus.DRAWER, ::DrawerScreen)
        MenuScreens.register(AdornMenus.KITCHEN_CUPBOARD, ::KitchenCupboardScreen)
        MenuScreens.register(AdornMenus.TRADING_STATION, ::TradingStationScreen)
        MenuScreens.register(AdornMenus.BREWER, ::BrewerScreen)
    }
}
